metricflow/test/plan_conversion/test_dataflow_to_sql_plan.py

```diff
@@ -1449,3 +1449,50 @@ def test_multi_hop_to_scd_dimension(
         sql_client=sql_client,
         node=dataflow_plan.sink_output_nodes[0].parent_node,
     )
+
+
+def test_multiple_metrics_no_dimensions(  # noqa: D
+    request: FixtureRequest,
+    mf_test_session_state: MetricFlowTestSessionState,
+    dataflow_plan_builder: DataflowPlanBuilder[DataSourceDataSet],
+    dataflow_to_sql_converter: DataflowToSqlQueryPlanConverter[DataSourceDataSet],
+    sql_client: SqlClient,
+) -> None:
+    dataflow_plan = dataflow_plan_builder.build_plan(
+        MetricFlowQuerySpec(
+            metric_specs=(MetricSpec(element_name="bookings"), MetricSpec(element_name="listings")),
+            time_range_constraint=TimeRangeConstraint(
+                start_time=as_datetime("2020-01-01"), end_time=as_datetime("2020-01-01")
+            ),
+        )
+    )
+
+    convert_and_check(
+        request=request,
+        mf_test_session_state=mf_test_session_state,
+        dataflow_to_sql_converter=dataflow_to_sql_converter,
+        sql_client=sql_client,
+        node=dataflow_plan.sink_output_nodes[0].parent_node,
+    )
+
+
+def test_metric_with_measures_from_multiple_sources(  # noqa: D
```

Comment:
Can we put the fact that there are no groupbys in this test name somewhere? You'll have to move all the plan files around.